modules/ldap_groups/app/views/ldap_groups/synchronized_groups/new.html.erb

```diff
@@ -54,7 +54,7 @@
 
     <div class="form--field -required">
       <%= f.select :group_id,
-                   Group.pluck(:lastname, :id),
+                   Group.order(Arel.sql("lastname ASC")).pluck(:lastname, :id),
```

Comment:
This can be written:

```suggestion                                      Group.order(:lastname, :asc).pluck(:lastname, :id),
```

There's no need to use Arel/raw SQL here.